requirements.txt

```diff
@@ -5,3 +5,4 @@ colorlog~=2.7
 libclang-py3~=3.4.0
 Pygments~=2.1
 PyPrint~=0.2.6
+dependency_management==0.1.5
```

Comment:
be sure to make a commit to coala to remove the requirement there as it's implicitly in coala from the next prerelease! Can you make an issue